VALUES.md

```diff
@@ -263,21 +321,62 @@ declarations in most cases.  Here are the rules, applied in order:
 - If the value is a function with a single argument `unit -> τ2`, then
   the declaration is assumed to be a `[@@js.get]` global getter.
 
-- If the value is a function with two arguments `τ1 -> τ2 -> unit` and
+- If the value is a function whose first argument is a named type `τ
+  -> ...`, then the definition is assumed to be a `[@@js.call]` method
+  call.
+
+- Otherwise, the declaration is assumed to be a `[@@js.global]` value.
+  This applies in particular for any non-functional type.
+
+### Rules for the *static* mode
```

Comment:
Is `static` mode strictly needed? What is it adding that the auto mode can't do?